roles/client-apt-remove/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Remove specified packages. "with_items"-method depreciated. Warning: "This feature will be removed in version 2.11"
+
+- name: Remove unwanted packages
+  apt:
+    pkg: ['pdfsam', '']
```

Comment:
Was hälst du davon, das aus einer Variable zu lesen? Die kann man dann in den Group_vars oder host_vars setzen.

Reply:
gute Idee!